examples/scrollbar.rs

```diff
@@ -9,7 +9,7 @@ use crossterm::{
     execute,
     terminal::{disable_raw_mode, enable_raw_mode, EnterAlternateScreen, LeaveAlternateScreen},
 };
-use ratatui::{prelude::*, widgets::*};
+use ratatui::{prelude::*, symbols::scrollbar, widgets::*};
```

Comment:
Why doesn't `widgets::scrollbar` conflict with `symbols::scrollbar`?

Reply:
This is because widgets::scrollbar is no longer re-exported, so there is no widgets::scrollbar module.

Before:

<img width="475" alt="image" src="https://github.com/tui-rs-revival/ratatui/assets/381361/816aef29-df78-42bb-a940-3ccd86f95258">

After:

<img width="516" alt="image" src="https://github.com/tui-rs-revival/ratatui/assets/381361/9f605672-261f-4c52-b415-fcbbbd2da603">